core/src/test/java/org/apache/calcite/rex/RexProgramTest.java

```diff
@@ -2458,6 +2458,79 @@ private SqlOperator getNoDeterministicOperator() {
     checkSimplifyUnchanged(isNull(cast(vVarcharNotNull(), tInt(false))));
   }
 
+  /** Unit test for
```

Comment:
I noticed that there are many duplicated java doc for those tests, it would be much better if we propose a suitable way to organize them. But I'm OK with it if remains still.